src/mango/src/mango_execution_stats.hrl

```diff
@@ -16,5 +16,6 @@
     totalQuorumDocsExamined = 0,
     resultsReturned = 0,
     executionStartTime,
-    executionTimeMs
+    executionTimeMs,
+    dbname
```

Comment:
Since all the other fields have full-length names, `databaseName` might be a better fit here for consistency.

Reply:
Hmmm... I went with `dbname` because it's the primary name used in the codebase to track the name of the db. This module here is the exception, we don't normally utilize camel case in record definitions, so I'm not a big fan of spelling out `databaseName`.